flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer010.java

```diff
@@ -173,7 +205,38 @@ public FlinkKafkaConsumer010(Pattern subscriptionPattern, DeserializationSchema<
 	 */
 	@PublicEvolving
 	public FlinkKafkaConsumer010(Pattern subscriptionPattern, KafkaDeserializationSchema<T> deserializer, Properties props) {
-		super(subscriptionPattern, deserializer, props);
+		this(null, subscriptionPattern, deserializer, props);
```

Comment:
Can we avoid `null` and instead pass in an empty collection? If not, then let's add `@Nullable` annotation.

Reply:
The `KafkTopicsDescriptor` actually relies on this being null. I'll add Nullable to the private constructor for the time being.